service/kas/access/rewrap.go

```diff
@@ -75,49 +86,64 @@ func generateHMACDigest(ctx context.Context, msg, key []byte) ([]byte, error) {
 	return mac.Sum(nil), nil
 }
 
-func verifySignedRequestToken(ctx context.Context, in *kaspb.RewrapRequest) (*RequestBody, error) {
+func noverify(ctx context.Context, srt string) (string, error) {
+	token, err := jwt.Parse([]byte(srt), jwt.WithValidate(false))
```

Comment:
I haven't really thought about this but should we still validate the srt?

Reply:
We don't really need to however we are signing it and I'd like to continue supporting this functionality until we decide to remove it explicitly with an ADR